### PR TITLE
ffmpeg-qsv: add wa_ldb to allow b_strategy disable

### PR DESCRIPTION
**Pull request by yefengxx**
ffmpeg-qsv: add wa_ldb to allow b_strategy disable

Signed-off-by: Xu Yefeng <yefengx.xu@intel.com>

### CODE REVIEW

**Comment by yefengxx** [comment]
@uartie @FocusLuo can you review this for me?

**Review by uartie — APPROVED** [review]

**Comment by uartie** [comment]
@yefengxx please update commit message to reflect the change made.  No need to mention "quality".  How about `ffmpeg-qsv: add wa_ldb to allow b_strategy disable`

**Review by uartie — CHANGES_REQUESTED** [review]
commit title

**Comment by yefengxx** [comment]
> ffmpeg-qsv: add wa_ldb to allow b_strategy disable

thanks for your advice.

**Comment by yefengxx** [comment]
> @yefengxx please update commit message to reflect the change made. No need to mention "quality". How about `ffmpeg-qsv: add wa_ldb to allow b_strategy disable`

thanks for your advice.

**Comment by uartie** [comment]
> > @yefengxx please update commit message to reflect the change made. No need to mention "quality". How about `ffmpeg-qsv: add wa_ldb to allow b_strategy disable`
> 
> @uartie thanks for your advice.

Please update the commit message, not just the PR title.

File: lib/ffmpeg/encoderbase.py [hunk 1]
```diff
@@ -89,7 +89,8 @@ def gen_output_opts(self):
 
     # WA: LDB is not enabled by default for HEVCe on gen11+, yet.
     if get_media()._get_gpu_gen() >= 11 and self.codec.startswith("hevc"):
-      opts += " -b_strategy 1"
+      if vars(self).get("wa_ldb", True):
+        opts += " -b_strategy 1"
```
Comment (uartie) [inline lib/ffmpeg/encoderbase.py:93]: @FocusLuo @feiwan1 is this workaround still needed, or can we delete it now?

Comment (uartie) [reply lib/ffmpeg/encoderbase.py:93]: removed in #433

**Pull request closed without merging**